src/tree/utils/walk.ts

```diff
@@ -49,10 +49,24 @@ export function* processNode(node: JSONSchema4): IterableIterator<SchemaNode> {
       yield combinerNode;
     }
   } else if (type) {
+    const primaryType = getPrimaryType(node);
+    let validationNOde: JSONSchema4 = node;
```

Comment:
is there a reason this is `validationNOde` and not `validationNode`?

Reply:
Ah yeah. Typo.